IFC4x3/Properties/a/AreaID_1LX3rTqy19RRHx3VNZyWTI/Documentation.md

```diff
@@ -0,0 +1 @@
+Indication of the single, specific MappedUnit in the same Mapped Zone.
```

Comment:
What is `MappedUnit`? What is `Mapped Zone`? Why are the formatted differently? It seems to me that they are not IFC based identifiers. Could we get a refence?

Reply:
I would write "Identification of the single, specific mapped feature in the same mapped zone."
And also add a note: ">NOTE Mapped zone is represented in IFC using _IfcSpatialZone_ using PredefinedType _MAPPEDZONE_."
I would also change datatype to IfcIdentifier as @SergejMuhic suggests below.
Ping @j-weil

Reply:
IfcIdentifier sounds reasonable to me